pyproject.toml

```diff
@@ -0,0 +1,66 @@
+# This file is protected via CODEOWNERS
```

Comment:
Let's remove this file. The changes that were made to this file in https://github.com/urllib3/urllib3/pull/2701 should be done inside of `setup.py` and `setup.cfg`.

Reply:
fixed in new commit